README.md

```diff
@@ -436,22 +440,50 @@ Built-in functions. Ready to be bound to any key you like. :smile:
-|------------------------------------|-----------------------------------------------------------------|
-| `previewers.cat.new`               | Default previewer for files. Uses `cat`/`bat`                   |
-| `previewers.vimgrep.new`           | Default previewer for grep and similar. Uses `cat`/`bat`        |
-| `previewers.qflist.new`            | Default previewer for qflist. Uses `cat`/`bat`                  |
-| `previewers.vim_buffer_cat.new`        | Experimental previewer for files. Uses vim buffers              |
-| `previewers.vim_buffer_vimgrep.new`    | Experimental previewer for grep and similar. Uses vim buffers   |
-| `previewers.vim_buffer_qflist.new`     | Experimental previewer for qflist. Uses vim buffers             |
-| .................................. | Your next awesome previewer here :D                             |
+| Previewers                          | Description                                                     |
+|-------------------------------------|-----------------------------------------------------------------|
+| `previewers.cat.new`                | Default previewer for files. Uses `cat`/`bat`                   |
+| `previewers.vimgrep.new`            | Default previewer for grep and similar. Uses `cat`/`bat`        |
+| `previewers.qflist.new`             | Default previewer for qflist. Uses `cat`/`bat`                  |
+| `previewers.vim_buffer_cat.new`     | Experimental previewer for files. Uses vim buffers              |
+| `previewers.vim_buffer_vimgrep.new` | Experimental previewer for grep and similar. Uses vim buffers   |
+| `previewers.vim_buffer_qflist.new`  | Experimental previewer for qflist. Uses vim buffers             |
+| ..................................  | Your next awesome previewer here :D                             |
 
 By default, telescope.nvim uses `cat`/`bat` for preview. However after telescope's new experimental previewers
 are stable this will change. The new experimental previewers use tree-sitter and vim buffers, provide much
-better performance and are ready for daily usage (`vim_buffer_cat` more than the others), but there might be
-cases where it can't detect a Filetype correctly, thus leading to wrong highlights.
-Also `vimgrep` and `qflist` might be slower due to synchronous file loading.
-
+better performance and are ready for daily usage, but there might be cases where it can't detect a Filetype
+correctly, thus leading to wrong highlights. This is because we can't determine the filetype in the traditional way
+(we don't do `bufload`. We read the file async with `vim.loop.fs_` and attach only a highlighter), because we can't
+execute autocommands, otherwise the speed of the previewer would slow down considerably.
+We determine the filetype by the extension. You can add more extensions to the filetypeguesser:
+Create file `~/.config/nvim/data/plenary/filetypes/builtin.lua` and add following:
```

Comment:
This should just link to some place in plenary where we show people how to do this.

This is the wrong repo. Perhaps we can add it in a help doc for plenary and say: `see ":help plenary-filetype"` or similar.

Reply:
Tami didn't like that here as well. People were just asking over and over and we need a place to document that :)